src/components/application_manager/test/mock_message_helper.cc

```diff
@@ -347,6 +350,41 @@ const VehicleData& MessageHelper::vehicle_data() {
   return MockMessageHelper::message_helper_mock()->vehicle_data();
 }
 
+bool MessageHelper::SendStopAudioPathThru(ApplicationManager& app_mngr) {
+  return MockMessageHelper::message_helper_mock()->SendStopAudioPathThru(
+      app_mngr);
+}
+
+std::string MessageHelper::StringifiedFunctionID(
```

Comment:
@AGaliuzov, `StringifiedFunctionID` can return temporary string and it can't take from it reference.